cfgov/unprocessed/css/organisms/footer.less

```diff
@@ -41,9 +43,9 @@
     }
 
     &_nav-list {
-        .list__links();
+        .m-list__links();
```

Comment:
So this changes from `m-list__links()` to `m-list__horizontal` based on the screen size, so that's why it's in here as a mixin.